apps/resources/scala-experimental.json

```diff
@@ -0,0 +1,34 @@
+{
+  "repositories": [
+    "central"
+  ],
+  "dependencies": [
+    "org.virtuslab.scala-cli:cli_3:latest.release"
+  ],
+  "launcherType": "graalvm-native-image",
+  "prebuiltBinaries": {
+    "x86_64-pc-linux": "gz+https://github.com/VirtusLab/scala-cli/releases/download/v${version}/scala-cli-x86_64-pc-linux.gz",
+    "x86_64-pc-win32": "zip+https://github.com/VirtusLab/scala-cli/releases/download/v${version}/scala-cli-x86_64-pc-win32.zip",
+    "x86_64-apple-darwin": "gz+https://github.com/VirtusLab/scala-cli/releases/download/v${version}/scala-cli-x86_64-apple-darwin.gz",
+    "aarch64-apple-darwin": "gz+https://github.com/VirtusLab/scala-cli/releases/download/v${version}/scala-cli-aarch64-apple-darwin.gz"
+  },
+  "name": "scala",
+  "versionOverrides": [
+    {
+      "versionRange": "[0.0.1,0.1.2]",
+      "repositories": [
+        "central",
+        "sonatype:snapshots"
```

Comment:
No need to add this one, as the versions it has don't have pre-built binaries on GitHub. 
```suggestion
```

Reply:
Just saw that it was in the original descriptor… https://github.com/coursier/apps/pull/185 should fix it.

Reply:
Removed!